src/Components/Filters/Filters.js

```diff
@@ -0,0 +1,139 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { useProducts } from "../../contexts/product-context";
+
+export const Filters = () => {
+  const ratingOptions = ["4", "3", "2", "1"];
+  const priceOptions = ["0", "5000", "10000", "15000", "20000", "25000", "30000", "35000", "40000", "45000", "50000", "55000", "60000", "65000", "70000", "75000", "80000", "85000", "90000"];
+  
+  const {
+    state: { sortBy, rating, categories, priceRange },
+    dispatch,
+  } = useProducts();
+
+  const [category, setCategory] = useState([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/categories");
+        setCategory(response.data.categories);
+      } catch (error) {
+        console.error(error.message);
```

Comment:
rather than showing error on console you can use react toastify

Reply:
Sure, Thanks